### ISSUE TITLE
Filtered list of pull requests (by label)

### ISSUE THREAD

**Issue by DennisSchiefer** [github/hub#2084]
I would like to be able to retrieve a list of pull requests filter by label.

This should be able to be combined with the filter for state.

**Comment by mislav** [comment]
I agree that this should be possible. See https://github.com/github/hub/issues/2079#issuecomment-472507867 for explanation why we would need significant refactoring and switch to a different API to achieve this, and an example of how you can search manually via `hub api` in the meantime.

Also, you can use `hub issue --include-pulls`, but that also includes issues among pull requests of course.
